protos/sources/vega/commands/v1/commands.proto

```diff
@@ -415,3 +415,15 @@ message JoinTeam {
   // ID of the team to join, this is the same as the referral code used to generate the team.
   string id = 1;
 }
+
+// Command to associate metadata to a public key, known as a party ID.
+// Partial update is not supported, meaning previous values must be set to be kept,
+// otherwise, they are voided.m
```

Comment:
```suggestion
// Partial update is not supported, meaning previous values must be set to be kept,
// otherwise, they are voided.
```

Reply:
And a follow-up question about what actually gets voided here, it wasn't clear to me.